app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/UserSetting.kt

```diff
@@ -7,21 +7,21 @@ import com.google.gson.annotations.SerializedName
  * Created by meeera on 30/6/17.
  */
 
-data class UserSetting (
+data class UserSetting(
 
-    /**
-     * Gets session
-     *
-     * @return the session
-     */
-    @SerializedName("session")
-    val session: Session? = null,
+        /**
+         * Gets session
+         *
+         * @return the session
+         */
+        @SerializedName("session")
+        val session: Session? = null,
 
-    /**
-     * Gets settings
-     *
-     * @return the settings
-     */
-    @SerializedName("settings")
-    val settings: Settings? = null
+        /**
+         * Gets settings
+         *
+         * @return the settings
+         */
+        @SerializedName("settings")
+        val settings: Settings? = null
```

Comment:
Indentation problem

Reply:
@iamareebjamal I reindented this code, but it remains the same. These spaces appear like this probably because the * is not aligned vertically with @.

Reply:
OK